Cargo.toml

```diff
@@ -24,16 +24,17 @@ combine = "3.8.1"
 bytes = "0.5"
 futures-util = { version = "0.3.0", features = ["sink"], default-features = false }
 futures-executor = "0.3.0"
-pin-project-lite = "0.1"
+pin-project-lite = { version = "0.1", optional = true }
 tokio-util = { version = "0.2", features = ["codec"] }
-tokio = { version = "0.2", features = ["tcp", "uds", "io-util", "sync", "stream"] }
+tokio = "0.2"
```

Comment:
Can't `tokio` be an optional dependency?

Reply:
Ditto the other `tokio` and `futures` crates

Reply:
I tried to get rid of it, but `BufRead` and the likes are required internally :/

Reply:
Right, forgot that synchronous parsing forwarded to the async ones. It can be fixed once I release combine 4.0 https://github.com/Marwes/combine/pull/279